src/core/utils/index.js

```diff
@@ -224,6 +224,16 @@ var utils = module.exports = {
     },
 
     /**
+     * Retunrs sign of number
+     *
+     * @param n {number}
+     * @returns {number} 0 if n is 0, -1 if n is negative, 1 if n i positive
+     */
+    sgn: function (n){
```

Comment:
Perhaps just call the function *sign*, or *getSign*. Don't see the need to abbreviate it by just one character.